HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -694,6 +694,25 @@ def test_ventilation_fans
     _test_default_bath_fan_values(hpxml_default, 2, 50, 1, 15, 7)
   end
 
+  def test_clothes_dryer_exhaust
+    # Test inputs not overridden by defaults
+    hpxml_name = 'base-appliances-gas.xml'
+    hpxml = HPXML.new(hpxml_path: File.join(@root_path, 'workflow', 'sample_files', hpxml_name))
+    clothes_dryer = hpxml.clothes_dryers[0]
+    clothes_dryer.is_vented = true
```

Comment:
Since `is_vented` defaults to true, the defaulting of this parameter is not actually being tested. I suggest an additional test where you set `is_vented = false` and ensure it stays false (i.e., not overridden by default and changed to true).